src/win/util.c

```diff
@@ -1466,6 +1466,59 @@ int uv_os_setpriority(uv_pid_t pid, int priority) {
   return r;
 }
 
+int uv_thread_getpriority(uv_thread_t tid, int* priority) {
+  int r;
+
+  if (priority == NULL)
+    return UV_EINVAL;
+
+  if (pGetThreadPriority != NULL) {
+    r = pGetThreadPriority(tid);
+  } else {
+    r = GetThreadPriority(tid);
+  }
```

Comment:
Should be something like:
```suggestion
  if (pGetThreadPriority == NULL)
    return UV_ENOSYS;

  r = pGetThreadPriority(tid);
```

Reply:
Since the error code 0xc0000135 is caused by the CI issue, maybe I should revert the change that 'retrieve GetThreadPriority and SetThreadPriority from kernel32.dll' ?

Because if I retrieve them from kernel32.dll, pGetThreadPriority will always be NULL on my computer. And I got the following error for the test case.


C:\Users\casa\Desktop\libuv\build\Debug>uv_run_tests_a thread_priority
not ok 1 - thread_priority
# exit code 3
# Output from process `thread_priority`:
# Assertion failed in C:\Users\casa\Desktop\libuv\test\test-thread-priority.c on line 62: `uv_thread_getpriority(task_id, &priority) == 0` (-4054 == 0)

Reply:
Yes, fair enough.

Reply:
Updated